share/commands/agent-check

```diff
@@ -248,7 +248,7 @@ check_and_fix_inputs() {
   fi
 
   # Ensure main promises are not broken, and clean ncf cache and force a new download if so
-  if ${CFE_BIN_DIR}/cf-promises | grep error > /dev/null
+  if ! ${CFE_BIN_DIR}/cf-promises > /dev/null
```

Comment:
Are you sure it returns !=0 when it fails

Reply:
yes, on my tests that's the case

Reply:
at least on similares version of rudder